features/support/poor_mans_webmock.rb

```diff
@@ -9,8 +9,14 @@ def perform(_sender_atom_url)
     end
   end
 
-  class HttpMulti < Base
-    def perform(_user_id, _enc_object_xml, _person_ids, _retry_count=0)
+  class SendPrivate < Base
+    def perform(_sender_id, _obj_str, _targets, _retry_count=0)
```

Comment:
What do you think about `def perform(*_args)`? This would reduce the amount of changes we need to do if we ever change the signature of this function and we have other specs to check if the functions are called correctly.